src/Compilers/VisualBasic/Portable/Analysis/FlowAnalysis/AbstractFlowPass.vb

```diff
@@ -339,14 +331,31 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                 Return False
             End If
             Select Case type.SpecialType
-                Case SpecialType.None,
-                     SpecialType.System_Nullable_T,
-                     SpecialType.System_IntPtr,
-                     SpecialType.System_UIntPtr
-                    Return True
+                Case SpecialType.System_Void,
```

Comment:
Flipped return values to avoid `SpecialType.None`. Please double-check this list